ServerCore/Pages/Submissions/Index.cshtml.cs

```diff
@@ -31,26 +32,56 @@ public IndexModel(ServerCore.DataModel.PuzzleServerContext context)
 
         public async Task<IActionResult> OnPostAsync(int puzzleId, int teamId)
         {
+
+            // Don't allow submissions after the answer has been found.
+            if (AnswerToken != null)
```

Comment:
AnswerToken is a weird thing to key off. Should be the PuzzleStatePerTeam.SolvedTime property..